### PR TITLE
change audit keystore to a pid

### PR DESCRIPTION
**Pull request by acdemyers**

### CODE REVIEW

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#run-libby-bot

File: dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -39,13 +39,15 @@ encrypt.desc=Indicates whether the audit records in the audit file log will be e
 encryptAlias.name=Alias name of certificate for encrypting
 encryptAlias.desc=Alias name of the certificate used to encrypt the audit records.
 
-encryptKeyStoreRef.name=Audit encryption keystore reference 
+encryptKeyStoreRef.name=Audit Encryption Keystore Reference 
+encryptKeyStoreRef.name$Ref=Audit encryption keystore reference
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties:43]: suggested edit:

Location of the keystore that contains the certificate that is used to encrypt the audit records.

File: dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -39,13 +39,15 @@ encrypt.desc=Indicates whether the audit records in the audit file log will be e
...
 encryptKeyStoreRef.desc=Location of the keystore containing the certificate used to encrypt the audit records.
 
 signingAlias.name=Alias name of personal certificate for signing
 signingAlias.desc=Alias name of the personal certificate used to sign the audit records.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties:47]: suggested edit

Alias name of the personal certificate that is used to sign the audit records.

File: dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties [hunk 3]
```diff
@@ -39,13 +39,15 @@ encrypt.desc=Indicates whether the audit records in the audit file log will be e
 encryptAlias.name=Alias name of certificate for encrypting
 encryptAlias.desc=Alias name of the certificate used to encrypt the audit records.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties:40]: Suggested edit:
Alias name of the certificate that is used to encrypt the audit records.

**Review by dmuelle — COMMENTED** [review]
The parts of the file that are being updated look fine, but the description fields here and throughout the file need edits per ID guidance, see my suggestions inline. If that text is beyond the scope of this PR I can approve the sections that are updated

File: dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties (hunk 3 above)
Comment (acdemyers) [reply dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties]: Message updated.  Please take a look.

File: dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties (hunk 2 above)
Comment (acdemyers) [reply dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties]: Message updated.  Please take a look.

**Comment by acdemyers** [comment]
#build

**Review by dmuelle — APPROVED** [review]
changes look good- there is one more description at line 44 that needs an update to replace "containing" with "that contains"

**Comment by acdemyers** [comment]
#run-libby-bot

**Review by wrodrig — APPROVED** [review]

**Comment by acdemyers** [comment]
#run-libby-bot

**Comment by acdemyers** [comment]
#build

**Pull request merged**